client/joinus_client/src/components/UserPage/AddEvent.js

```diff
@@ -12,7 +12,7 @@ import {
   styled,
   TextField,
   Tooltip,
-  Typography
+  Typography,
```

Comment:
Oh weird it added on here too - I wonder if a linter setting is causing this? /shrug